test/Integration/tpp-pack.mlir

```diff
@@ -0,0 +1,63 @@
+// RUN:  tpp-opt %s  --default-tpp-passes | tpp-run -e entry  -entry-point-result=void | FileCheck %s
```

Comment:
No need for `tpp-opt` here, just `tpp-run`.

Reply:
Not having tpp-opt throws an error : Entry point entry not found as a func.func
Did you mean get rid of the flag -default-tpp-passes?

Reply:
You should be able to run this with just `tpp-run`. If you drop the `-default-tpp-passes` from `tpp-opt` then it does nothing.